test/gcs/helper_conn_test.go

```diff
@@ -0,0 +1,324 @@
+//go:build linux
+
+package gcs
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+
+	"github.com/Microsoft/hcsshim/internal/guest/stdio"
+	"github.com/Microsoft/hcsshim/internal/guest/transport"
+)
+
+const (
+	_network     = "unix"
+	_sockPathFmt = "/tmp/gcs.%d"
+
+	_dialRetries = 4
+	_dialWait    = 50 * time.Millisecond
```

Comment:
What is the underscore indicating here?

Reply:
opps, they shouldn't be there for the constants; I add the to global `vars` so I dont accidentally overwrite them with locals of the same name.